.claude/skills/mock-to-unit/instructions.md

```
@@ -0,0 +1,259 @@
+## Phase 2: Conversion
+
+When running conversion mode:
+
+1. **Read the target mock test file**
+2. **Identify the components under test**:
+   - Look at page object imports
+   - Trace back to actual component files
+   - Find component file paths (e.g., `src/components/deploy/DeploymentForm.tsx`)
+
+3. **Check for existing unit tests**:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)